node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,9 @@
+import { writeFile } from 'node:fs/promises';
+
+const argv = process.argv[2] + '\n';
```

Comment:
`note` would be a more descriptive variable name. `argv` doesn't have any contextual meaning.